tree/30_generic_methods/_classes_success.cf

```diff
@@ -38,6 +39,10 @@ bundle agent _classes_success(class_prefix)
       "local_${suffix}"
         string     => "${class_prefix}_${suffix}",
         classes    => always("${class_prefix}_${suffix}");
+
+  classes:
+      "destination_exists" not => strcmp("${class_prefix}", "");
```

Comment:
In master we use this property in every generic method so that calls don't fail when this.caller_promisers doesn't exist. So it could be a warning, but not in the general case.